views/js/pciCreator/dev/audioRecordingInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -34,10 +35,10 @@
 
 <div data-role="delayOptions" {{#unless autoStart}}style="display:none"{{/unless}}>
     <div class="panel">
-        <label for="delayMinutes" class="spinner">{{__ "Min:"}}</label>
-        <input name="delayMinutes" value="{{delayMinutes}}" data-increment="1" data-min="0" data-max="20" type="text" />
-        <label for="delaySeconds" class="spinner">{{__ "Sec:"}}</label>
-        <input name="delaySeconds" value="{{delaySeconds}}" data-increment="15" data-min="0" data-max="45" type="text" />
+        <label for="delay-minutes" class="spinner">{{__ "Min:"}}</label>
+        <input id="delay-minutes" name="delayMinutes" value="{{delayMinutes}}" data-increment="1" data-min="0" data-max="20" type="text" />
```

Comment:
I suppose `id` is used for data binding. 
Please check here https://github.com/oat-sa/extension-tao-itemqti-pci/blob/ffc523d6ba1bc7613c9f60cdc711ca8d4f62cd96/views/js/pciCreator/dev/audioRecordingInteraction/creator/widget/states/Question.js#L80-L102

Reply:
There wasn't an ID prior to this PR ;)

The data binding is intact.

Reply:
Sorry, my bad :)